src/Slave.cc

```diff
@@ -5,6 +5,29 @@
 
 namespace kickcat
 {
+    void printGeneralEntry(eeprom::GeneralEntry const* ge) {
```

Comment:
linefeed and variable name is not meaningful (prefer general_entry or at least entry)